socorro/processor/rules/memory_report_extraction.py

```diff
@@ -0,0 +1,132 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from socorro.lib.transform_rules import Rule
+
+
+UNITS_BYTES = 0
+KIND_NONHEAP = 0
+KIND_HEAP = 1
```

Comment:
`KIND_NONHEAP` and `KIND_HEAP` appear to be special values in the memory report. Is there a definition of those values or a specification somewhere? If there is, I'd toss the link in the module docstring or something like that so it's easier to find later.